pages/languages/javascript.md

```diff
@@ -0,0 +1,40 @@
+---
+permalink: /languages/javascript/
+title: JavaScript
+parent: Languages
+---
+
+# JavaScript Ecosystem Guide
+
+The purpose of this guide is to help familiarize yourself with the JavaScript ecosystem and set up a handful of tools to help create a seamless working development environment. In particular, we’ll be using a tool called `nvm` to make sure we’ve got the right versions of `node` and `npm` installed for a project.
+
+## Node Installation
+
+Node is super easy to install and you can do so in a number of ways. The fine folks behind the project have put a bunch of time and energy into ensuring cross platform compatibility and it totally shows.
+
+You can install it through the binaries on nodejs.org, through `brew` on Mac OS X, or compiling it from source but the preferred method is through a tool called `nvm`. `nvm` stands for Node Version Manager. It will enable us to easily install and use different versions of Node (including the relatively short lived iojs fork).
+
+## nvm Installation
+
+You can find the homepage for the project at https://github.com/creationix/nvm and [installation instructions](https://github.com/creationix/nvm#install-script) there. Overall, it is a really easy tool to set up, and can generally done with copying and pasting the following line of code into your terminal:
+curl -o- https://raw.githubusercontent.com/creationix/nvm/v0.29.0/install.sh | bash
```

Comment:
Great point, Marco. I think we can add a reference to installing on Windows and point to that library. It looks pretty good and was updated somewhat recently.